app/controllers/delivery_partners/participants_controller.rb

```diff
@@ -62,5 +62,9 @@ def to_csv(hash)
         end
       end
     end
+
+    def filter_params
+      params.permit(:query, :role, :academic_year, :status).merge(delivery_partner:)
```

Comment:
I think we can permit the `delivery_partner_id` instead as it's already available? to avoid merging in an object to params?

Reply:
nvm will destroy the serialiser performance otherwise